src/lib/containers/CopyToClipboardInput.tsx

```diff
@@ -10,10 +10,10 @@ export type CopyToClipboardInputProps = {
 
 /**
  * Component that holds a large string in a readonly <input> to be copied to the user's clipboard when clicked.
- * This component should only be used when the value to copy is not very long AND the full length of the string
```

Comment:
Removed `value to copy is not very long` from the comment since it is more confusing than helpful.